src/ui/components/modal/HelpModal.tsx

```diff
@@ -12,71 +12,83 @@ export const HelpModal = ({ isOpen, setIsOpen }: Omit<ModalProps, 'title'>) => {
   return (
     <Modal isOpen={isOpen} setIsOpen={setIsOpen} title="Help">
       <ul className="text-gray-300 text-sm">
-        <li className="flex w-full justify-between items-center border-b border-gray-800">
-          <div className="flex py-4">
-            <div className="w-5 mr-2" style={{ position: 'relative', top: 2 }}>
-              <BookOpenIcon className="h-4 w-4 " aria-hidden="true" />
+        <li>
+          <a
```

Comment:
just changed the order of the anchor tag, so that the whole width is clickable and used `group-hover` to add hover state